src/runtime_src/doc/toc/xrt_native_apis.rst

```diff
@@ -97,7 +97,8 @@ The above code block shows
 .. code:: c++
       :number-lines: 10
            
-           auto device = xrt::device(0);
+           unsigned int dev_index = 0;
+           auto device = xrt::device(dev_index);
```

Comment:
Why `auto` here?
```c++
xrt::device device(dev_index);
```
or even clearer
```c++
xrt::device device { dev_index };
```

Reply:
Agreed with {}.  As I said before, personally I prefer auto and types to the right.  The code reads much better that way IMO.  But C++ is inconsistent in this way, which is evident on line 100.